.pre-commit-config.yaml

```diff
@@ -47,11 +47,11 @@ repos:
       - id: mypy
         exclude: ^setup.py|^tests/|^vendor/|^astacus/proto/
         additional_dependencies:
-          - types-PyYAML>=6.0.12.2
-          - types-requests>=2.28.11.4
-          - types-tabulate>=0.9.0.0
-          - types-urllib3>=1.26.25.3
-          - typing_extensions>=3.10.0
+          - types-PyYAML
```

Comment:
Why drop explicit versions if they are required?

Reply:
Thanks for catching that